src/controllers/user-controller.js

```diff
@@ -15,3 +17,19 @@ export const findAllUsers = (obj, args, { models }) => {
 export const findUserById = (_, { id }, { models }) => {
   return models.user.findByPk(id)
 }
+
+export const signUp = async (_, data, { models }) => {
+  const user = await models.user.findOne({ where: { username: input.username }}) 
+  if (!user){
+    user = await models.user.createUser({ data })
+    const secret = process.env.JWT_SECRET;
+    const token = jwt.sign({ sub: user.id }, secret, { exp: '10d' } );
```

Comment:
10 days for an access token is wide too much time. Take a look at this video: https://www.youtube.com/watch?v=SLc3cTlypwM.

Reply:
Hi Abel, in the requirements passed in this module says it's 10 days https://github.com/WeDevelop-ARG/full-stack-beginner-program/blob/master/content/block-4/jwt-details.md.
Do you think less time is more appropriate?